README.md

```diff
@@ -143,13 +143,21 @@ def hello(data, context):
     print(context)
 ```
 
```

Comment:
(Similar comment above) The above `data, context` is not a CloudEvent, it's a GCF event. We should change this section as such.